src/python/structurefinder/pyberny/__init__.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2023 NWChemEx-Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pluginplay as pp
+from friendzone.nwx2qcelemental import chemical_system_conversions
+from simde import EnergyNuclearGradientStdVectorD, TotalEnergy
+from berny import Berny, geomlib, optimize
+import chemist
+import qcelemental as qcel
+
+
+class GeomoptViaPyberny(pp.ModuleBase):
+
+    def __init__(self):
+        pp.ModuleBase.__init__(self)
+        self.satisfies_property_type(EnergyNuclearGradientStdVectorD())
+        self.satisfies_property_type(TotalEnergy())
+        self.description("Performs PyBerny optimization")
+        self.add_submodule(TotalEnergy(), "Energy")
+        self.add_submodule(EnergyNuclearGradientStdVectorD(), "Gradient")
+
+    def run_(self, inputs, submods):
+        e_pt = TotalEnergy()
+        g_pt = EnergyNuclearGradientStdVectorD()
+        mol, = e_pt.unwrap_inputs(inputs)
+        grad, = g_pt.unwrap_inputs(inputs)
```

Comment:
```suggestion
```
If you remove the `satisfies_property_type(EnergyNuclearGradientStdVectorD())` you won't want this line anymore either (also this gives you back the returns on this line are the chemical system and the point at which to take the derivative, not the gradient as your code suggests).